ui/src/app/dashboard/total-summary/total-summary.component.css

```diff
@@ -1,18 +1,31 @@
-.mat-card {
+.total-summary{
```

Comment:
I think it would make sense to apply the same display: flex styling here

Reply:
Accepted.